packages/haproxy/0.1.3/dataset/info/fields/base-fields.yml

```diff
@@ -0,0 +1,23 @@
+- name: data_stream.type
+  type: constant_keyword
+  description: Data stream type.
+- name: data_stream.dataset
+  type: constant_keyword
+  description: Data stream dataset.
+- name: data_stream.namespace
+  type: constant_keyword
+  description: Data stream namespace.
+
+- name: dataset.type
```

Comment:
We still have the `dataset` fields inside?

Reply:
We'll clean these fields after everything in terms of renaming is done.